examples/arckubernetes/variables.tf

```diff
@@ -0,0 +1,78 @@
+7dEAMQ5PySRBXWt1FBdn0d55onwZy29GgqHge33/6sn/QJv5Fvsf2peXPqqJcCcE
+O0XEK+LqOWS9ukAzMyxIsXmC0T80qvTizETt0Vrh93WVslIRfqLj0yZJnDiFlsJH
+H31zFU9SivnxtXTUgiOzi7HSPueHEGgTONmnswzyQWwAdTG0NaKaCqpTaCbL1x9O
+FHD8Zq4ubHLWaJoStfyrFVcm4Hi1KGZ2MX5vOnxJLbG4+86Y/80g1kjJQ/s9mfrF
+Zh4MiQ6FD7iCRmAzTaRbn+4P0ToWrw9KTFDaBupudfw0OVfxAQKCAQEA2aLgcKkg
+urH+dNelUupbgQiVn+ctdNbR2AJuDSTH0jOBq6bsaqOymtkBrNHoFBooNZGI9hK3
+zJ7BQHNI60gQ6QJmI8lfPle40s5U281ZKoEQ7g/5NXHjMTZIHnv3pQ5hsdr2510F
+sQkZY7epQjwWb+CIf2cupZop+7P3/RYy2KEY06w8O9P6zjM+HRXqzdW3xMBV9qEe
+sGtGVhoX7SI7tNe6OGEHkQ5nLC1xlEQ/56IH73z7MMik0Zk162IQ2tkW+rsOQUWV
+E0JoqG4E0YW0cRd3VBjsOr75WPMJkjNCxVI8HP0BrnoLoR9Y5NPp9npkbs0PZoSL
+92Cy+HGNOVvazwKCAQEA2K9HGhes30z4vF6D8V2aKt2cfwZW+BRBiwp5Cy1mGMJE
+DYs0uYKeCFyTsrWBiAUVgFi+0hArSSGCgBg+2hqwx/uycrsrqDm3m+35gQt9KnTh
+fC6vCKN/9jbFadvSnBLJgmh9xLOR3wUtL26LIRwLDkOGiQOLEsz0akmj2BYjFugl
+ASK6Q33FPXNqEGLVhRdvKe2jtV+gtt9e815nqmuFeW6BFX60E5AAbIMW2yBLyI5s
+vuTTJVzAaNtkn32HhW2m5W94NTCL/RVYJTIJebn2G+STfNtuvT7aPazT45MwtHJl
+vDexRBOs1VEu2CadT9af6JIa+ba1rLQbLY9lClsIgQKCAQEAzJr44FHQqFRggh6e
+mdYkUqnsaRlgveCcBbU7G3ZBudech/D1gMgHSNbNYVkyE3wH41zAY39zdpal3l7D
+5hapdzY/KlPODAFSjDVYyBBFTNVsQjOYOeupCQIcSTSIvI9Jv3i4+QiWKjsAuKPZ
+J6DwAxwXl5c83omSAMXQVgqBoPHLMm1Rb3m56BN6yMNrLTpIe1vLNYOCI5+mz5nW
+dqThdwH9W7W09GWTD5BloFuNrMnDJh7c1J89fKPKclGOwQCpPu2O2ER/g0zCsd5E
+/2nxGwVRtSl9JTmXmE1c6SryEipew24FH7AzsLWU0AR+h6/HLffxC5N/NwGdID84
+4vz9NwKCAQBm/2oczQQvYLcbUxIEuSl/Frg86RaN9ZLyNJg05++P9UfD6Fe3lo7o
+CplqaTyZG1GNIKIT1/u7V2Jg4SV/JxDA/rNkNfQaktTkGcpvvLZSaByGmuo4gP9i
+Wcac+xCxf3ypsGGYDoNYL227DiC0LFBVB2P73ONOnH3RyA9Jt7wWN2Lfl1JSooYP
+H71a7J/yE6GTnJIktMAALU1FAarlJxbbEdvqC8AtTFXnPqiuoDtlqc2nL73uFJih
+PyglZ9XzZpvek43gOqNQOcUcaRx38B08RPqWbvfEnQvAJVbS5jRQkdAqF477fR66
+h0xtEm3UdYCtXEXWTTMuVK5WicLNmVgBAoIBACF6xdGPpo+SD3KSU1YaJ2C149ev
+l6mLqAU2tZTMIOJLMT/gEA28Csrry7c123xbXGUBfQdw3mK978RxsIVwBZlrozK7
+XOKOA2Cu31PdztVM5woDZKqMgWUahaLVFu91Kn5th6MEIeU81mK/SkPi5jwQpv+D
+JqZSH/Eht2lAgBPK7pVRMYU0F/buPmSiVtZZfP9IJrtxgXl2irnpX+PCCn5RHvfS
+ZfClhGbVuFPVZLdjE7yU3cXG89tx4V1E4OXc+Shc642saf/q5ubp4fhip7hm17An
+0wnfRuCGgHZrTORtpWslNcezgUjLvdGAaK2cTx27s3/tCUTT96fvLS8XZF0=
+-----END RSA PRIVATE KEY-----
+EOT
+}
+
+variable "public_key" {
+  description = "The base64-encoded public certificate used by the agent"
+  default = "MIICCgKCAgEAuDZwKNah3q5LsCBmPfokvwV+cMU/Bv57DnI5KQtMDPDipAsgJhb9f0TFSoSXTA9bbG+nCQca0npf03Saes4saFxEy8hP9tpRcyYDZvYKpLP7AzXlOkvwf5QqNRFLKYT0VSX069UHhQcGOzYL0CZL2bTtlLREIQNNH1kFW5sjGgrpEsMWDuKFGou2iqgWQEna6p/O/inxcnWERLbrfU+IfstK3BCHnKUAsno0ERUh/LyQrYj+svJYESGaiHoHuP6picRrNDV5V1ntbkYTV17e/LOGSEocp/Gp9t+PZh8Rf8+ZjouCIdnJo+/lIX5XPsTigKwbbEtqBCLx3S+0qjti4Afx/IV4ZAB9KMyOjfSM+ELPU080tI7cIUut2JQDoiiyenQt+I8BG4Mt3tajg2fTmd8ivOIWM7sdP+R+Rl0joLcS5TqA1K1/8Y5Xn9VYw0RwGeVLxM+uQ80Z83AcYYWYntKwEl9/JFsiqWWpgNIp6gU1H8KOsHBzV+PizN6HeB5TdgURllUjCl49gDIfpZYHm+Mm2pFgw7/f1H+e1TXhBti6vcKd1q2smN/bgAosNn1St+Rd2xrn21xWZum6WIeiMDoD5908i659rgBx9xPiWtGLfGbS976WblpRMopY3pL/E2TdtQBxXbjqpo1moVVezdTwQasI5mKZEAiQ0qO0uk8CAwEAAQ=="
+}
```

Comment:
We shouldn't be uploading any private/public key pairs, these should be generated or provide guidance on how to generate in the example